File: C++/Level1/Find Kim.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string solution(vector<string> seoul) {
	const string NAME = "Kim";
	int kimsIndex = distance(seoul.begin(), find(seoul.begin(), seoul.end(), NAME));

	return  "김서방은 " + to_string(kimsIndex) + "에 있다";
}

File: Java/Level1/FindKim.java
/* 
	https://school.programmers.co.kr/learn/courses/30/lessons/12919?language=java
*/

import java.util.Arrays;

class Solution {
    public String solution(String[] seoul) {
        int index = Arrays.asList(seoul).indexOf("Kim");
        
        return String.format("김서방은 %d에 있다", index);
    }
}